### ISSUE TITLE
Tag new release

### ISSUE THREAD

**Issue by benatespina** [gajus/dindent#26]
Hi,

we're using this amazing library to make project that uses the `svg` elements. PR that fixes bugs related with this king of elements has been merged months ago, but it does not exist any tag that contains the #23 PR.

So, can you tag a new release that allows to use the last merged improvements? :)

**Comment by gajus** [comment]
I haven't used Composer for couple of years now. Is adding a git tag sufficient to create a new release?

**Comment by benatespina** [comment]
@gajus No, you have to add a new Git tag and then, you have to click `Update` button in Packagist. https://packagist.org/packages/gajus/dindent

**Comment by gajus** [comment]
I've created two tags:

* https://github.com/gajus/dindent/tree/2.3.0
* https://github.com/gajus/dindent/tree/v2.3.0

and hit Update, but nothing is changing. Anything else I should do?

**Comment by gajus** [comment]
I am guessing I need to update https://github.com/gajus/dindent/blob/master/composer.json#L3.

**Comment by gajus** [comment]
Nop. Even after updating the git tags and updating the composer.json, the packagist.org does not pick up the change.

**Comment by benatespina** [comment]
Sorry,
Composer does not use the `version` directive to determine the current tag. So, you can safely remove this directive from `composer.json` because it's useless in its workflow.

To make work, only you have to tag a new git release in GitHub and hit the packagist `Update` button. You do not need to do anything in your codebase to publish a new release in Packagist.

**Comment by benatespina** [comment]
@gajus is it something that can I help you?

**Comment by benatespina** [comment]
@gajus ping

**Comment by gajus** [comment]
> @gajus No, you have to add a new Git tag and then, you have to click Update button in Packagist. https://packagist.org/packages/gajus/dindent

I have already done that and Packagist does not pick up the new tag. There isn't much else I can do.

**Comment by Schleuse** [comment]
FIY, you can use the zip directly from github as a workaround:

 ## composer.json:
```json
...
"require" : {
	"gajus/dindent" : "2.3.0",
},
"repositories" : [{
	"type" : "package",
	"package" : {
		"name" : "gajus/dindent",
		"version" : "2.3.0",
		"dist" : {
			"url" : "https://github.com/gajus/dindent/archive/2.3.0.zip",
			"type" : "zip"
		}
	}
}]
...
```

**Comment by SharkMachine** [comment]
> I have already done that and Packagist does not pick up the new tag. There isn't much else I can do.
> 
> Packagist does not provide any debugging information in response to the Update the action. The network request results in 200.

That's because when you check the contents of `composer.json` for tags `v2.3.0` and `2.3.0`, the version is still 2.0.2: 

https://github.com/gajus/dindent/blob/df486a42dc092105100a48a6c8ae7aa7db1aa61a/composer.json#L3

Those tags get ignored as the version in `composer.json` doesn't match with the tag:

![image](https://user-images.githubusercontent.com/7488694/116991086-b4246a00-acdc-11eb-9d6a-fb583b523393.png)

 (I set this GitHub repository as a VCS repository in `composer.json` before running that command)
